app/assets/javascripts/discourse/app/components/uppy-backup-uploader.js

```diff
@@ -20,7 +36,7 @@ export default Component.extend(UppyUploadMixin, {
     return { skipValidation: true };
   },
 
-  uploadDone() {
-    this.done();
+  uploadDone(responseData) {
+    this.done(responseData.fileName || responseData.file_name);
```

Comment:
Would it be possible to do `{ fileName: file.name }` in the uppy _upload mixin so we can omit the `||`?

Reply:
You're right, fixing